browser/installer/package-manifest.in

```diff
@@ -146,7 +146,7 @@
 @BINPATH@/@DLL_PREFIX@mozavcodec@DLL_SUFFIX@
 #endif
 @RESPATH@/browser/blocklist.xml
-#ifdef XP_WIN
+#if defined(XP_WIN) || defined(XP_LINUX)
```

Comment:
As this (pull request) was not merged before distribution of Waterfox Classic `2019.10 (20191023220600)`, I posted something to Reddit for the AMO use case: 

https://www.reddit.com/comments/d79h2l/-/f5htjhn/